api/src/test/java/org/openmrs/api/ConceptServiceTest.java

```diff
@@ -1251,13 +1246,12 @@ public void saveConceptNameTag_shouldNotSaveAConceptNameTagIfTagExists() {
 		ConceptNameTag secondNameTag = new ConceptNameTag();
 		secondNameTag.setTag(tag);
 		
-		ConceptNameTag existingConceptNameTag = conceptService.saveConceptNameTag(secondNameTag);
-		
+		assertThrows(Exception.class, () -> conceptService.saveConceptNameTag(secondNameTag));
 		assertNull(secondNameTag.getId());
-		assertEquals(existingConceptNameTag.getId(), nameTag.getId());
```

Comment:
Why did you remove this line?
From reading it I feel like its supposed to show that the `existingConceptNameTag.getId()` did not change. Thats why I believe it should stay. Unless you convince me it should not :)

Reply:
the exception happen at with this method  :		
       
         ConceptNameTag existingConceptNameTag = conceptService.saveConceptNameTag(secondNameTag);

if it has to stay , it will look like this 

    assertThrows(Exception.class, () ->  assertEquals(conceptService.saveConceptNameTag(secondNameTag).getId(), nameTag.getId());

Reply:
Thats right but you already wrapped the `conceptService.saveConceptNameTag(secondNameTag)` in an assertThrows. The assertThrows allows to do assertions after it. That is the reason why for example `assertNull(secondNameTag.getId());` is even possible. So it should also be possible to keep `		assertEquals(existingConceptNameTag.getId(), nameTag.getId());` in.

this is how I expect it to look like

```java
	assertThrows(Exception.class, () -> conceptService.saveConceptNameTag(secondNameTag));
	assertNull(secondNameTag.getId());
	assertEquals(existingConceptNameTag.getId(), nameTag.getId());
```